lib/pbench/server/api/resources/query_apis/iteration_samples.py

```diff
@@ -171,10 +185,10 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
 
     def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
         """
-        Returns a non-empty list of aggregated unique values for keyword fields
-        in the 'benchmark', 'sample', 'iterations' and 'run' subdocuments of
-        result-data-sample index. The output only contains fields that has at
-        least 1 unique value.
+        Returns a stringified JSON object containing keyword/value pairs where
+        each key is the fully qualified dot-separated name of a keyword
+        (sub-)field and each value is a non-empty list of Elasticsearch
+        bucket keys which appear in that field.
```

Comment:
It's not obvious to me that the caller of this function knows or should care that the list contents are Elasticsearch bucket keys.  The point is that the elements in those lists are _values_ associated with the corresponding keys in documents in the database, such that those key/value pairs could be used to select or filter the document sets in a subsequent DB query.